internal/command/postgres/create.go

```diff
@@ -34,6 +34,10 @@ func newCreate() *cobra.Command {
 		flag.Region(),
 		flag.Org(),
 		flag.Detach(),
+		flag.Bool{
+			Name:        "enable-backup",
```

Comment:
Let's change this to `enable-backups` and update the description to indicate that it will include the provisioning of a new Tigris bucket.